app/admin/withdrawal.rb

```diff
@@ -0,0 +1,35 @@
+ActiveAdmin.register Withdrawal do
+  permit_params :shop_id, :amount, :approved
+
+  index do
+    selectable_column
+    id_column
+    column :shop_id
+    column :amount
+    column :approved
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  filter :id
+  filter :shop_id
+  filter :amount
+  filter :approved
+  filter :created_at
+  filter :updated_at
+
+  show :title => :id do
+    attributes_table do
+      row :id
+      row :shop_id
+      row :amount
+      row :approved
```

Comment:
You could just divide by 100 to get the decimal value. You can create a new method in the model called `display_amount` or something like that and then in that just do the division. You can then use that method to show the amount.